### PR TITLE
4.6 OCP on OSP release notes.

### PR DESCRIPTION
**Pull request by maxwelldb**
I'm pulling together 4.6 release notes. Let me know if what's up here seems anemic, inaccurate, or incomplete. @racedo @EricArrakis @iamemilio @Fedosin @pierreprinetti @mandre

Feel free to make a PR directly, too: https://github.com/maxwelldb/openshift-docs/blob/osp-4.6-rn/release_notes/ocp-4-6-release-notes.adoc

FYI @codyhoag @jeana-redhat

### CODE REVIEW

**Comment by maxwelldb** [comment]
+ @luis5tb

File: release_notes/ocp-4-6-release-notes.adoc [hunk 1]
```diff
@@ -57,6 +57,34 @@ link:https://access.redhat.com/support/policy/updates/openshift[Red Hat OpenShif
...
+==== Bare metal machines for clusters on {rh-openstack}
+
+Clusters on {rh-openstack-first} can now run compute and control plane machines on bare metal. 
+
+For more information, see the {product-title} on {rh-openstack} installation documentation.
+
+[id="ocp-4-6-installation-osp-availability-zones"]
+==== {rh-openstack} availability zones selection at installation
+
+You can now select {rh-openstack} availability zones while installing a cluster on {rh-openstack}.
```
Comment (luis5tb) [inline release_notes/ocp-4-6-release-notes.adoc:73]: perhaps worth to highlight they are "nova AZs"

Comment (maxwelldb) [reply release_notes/ocp-4-6-release-notes.adoc]: Done in https://github.com/openshift/openshift-docs/pull/26195/commits/d2b0ddda854770cd799bc00079e6f80f40c9ea38

**Comment by maxwelldb** [comment]
Merge should be held pending OSASINFRA-992 status.

/hold

**Comment by maxwelldb** [comment]
992 is resolved, but merge should still be held pending completion of other items.

**Comment by maxwelldb** [comment]
@codyhoag @jeana-redhat Are you able to review this? Don't want to step on your toes, here.

**Comment by codyhoag** [comment]
I'll take a look :+1:

File: release_notes/ocp-4-6-release-notes.adoc [hunk 2]
```diff
@@ -273,6 +273,20 @@ Deployment now supports xref:../installing/installing_bare_metal_ipi/ipi-install
 
 Deployment now performs introspection on nodes to ensure that nodes meet installation requirements instead of generating errors if they do not.
 
+[id="ocp-4-6-installation-osp-availability-zones"]
+==== {rh-openstack} availability zones selection at installation
+
+You can now select {rh-openstack} Compute (Nova) availability zones while installing a cluster on {rh-openstack}.
```
Comment (codyhoag) [inline release_notes/ocp-4-6-release-notes.adoc:279]: Perhaps for the first mention in this description, we can use `{rh-openstack-first}` attribute, just to help out those Ctrl+f'ing for "OpenStack".

**Review by codyhoag — COMMENTED** [review]
One minor suggestion; otherwise; LGTM!

File: release_notes/ocp-4-6-release-notes.adoc (hunk 2 above)
Comment (maxwelldb) [reply release_notes/ocp-4-6-release-notes.adoc:279]: It is done.

**Comment by codyhoag** [comment]
Sorry; will also need commit squashing too @maxwelldb

**Comment by maxwelldb** [comment]
Done and merged. Thanks!

**Pull request merged**